COMMITERS.rst

```diff
@@ -0,0 +1,193 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
File name should be 'COMMITTERS.rst" (double T)

Reply:
Done